challenges/2017-08-15-kata-bathroom-security/README.md

```diff
@@ -0,0 +1,4 @@
+# Running the challenge
```

Comment:
Could you put a description and source of the problem? You can take a look at the other challenges 💃